models/CsvImporter.php

```diff
@@ -146,6 +180,14 @@ protected function getAdditionAdapterOptions()
             ];
         }
 
+        $returnValue['onResourceImported'] = [];
+        $returnValue['onResourceImported'][] = function (core_kernel_classes_Resource $resource) {
+            $resource->editPropertyValues(
+                new core_kernel_classes_Property(UserRdf::PROPERTY_DEFLG),
+                $resource->getOnePropertyValue(new core_kernel_classes_Property(UserRdf::PROPERTY_UILG))->getUri()
+            );
+        };
```

Comment:
Maybe we can extract this callback to a separate function?
As I can see, we use it twice